backend/src/test/groovy/com/capstone/moneytree/service/DefaultUserServiceTest.groovy

```diff
@@ -111,7 +111,7 @@ class DefaultUserServiceTest extends Specification {
         userDao.save() >> user
 
         when:
-        User userResponse = userService.editUserProfilePicture(user, image, "test")
+        User userResponse = userService.editUserProfilePicture(user, image, "avatarURL")
```

Comment:
Since cases were added into DefaultUserService in editUserProfilePicture method, add a test for "coverPhotoURL" and "default" selection. Those paths are not covered by the tests.

Reply:
You are right, I am adding tests right now :-)